OMSimulator/PI_Controller.lua

```diff
@@ -128,10 +137,14 @@ exitOnError(status)
 -- status code: 0
 -- status code: 0
 -- Simulation
---   limiter.u: -10.334369122725
---   limiter.y: -10.334369122725
+--   limiter.u: -10.014508893657
+--   limiter.y: -10.014508893657
 -- status code: 0
 -- status code: 0
 -- status code: 0
+-- limiter.u is not equal
+-- limiter.y is not equal
```

Comment:
This is because of the comparison function. Actually the signals look good. I will try to improve the comparison function.